DAGs.md

```diff
@@ -261,8 +261,9 @@ ETL Process: Use the API to identify all CC-licensed images.
 
 Output: TSV file containing the image, the respective meta-data.
 
-Notes: https://freesound.org/apiv2/search/text' No rate limit specified. This
-script can be run either to ingest the full dataset or as a dated DAG.
+Notes: https://freesound.org/docs/api/ Rate limit: 60 requests per minute, 2000
+requests per day. This script can be run either to ingest the full dataset or as
+a dated DAG.
```

Comment:
This should not apply to us as Frederic Font from Freesound granted us unlimited permissions on our API key. Do these refer to a technical limitation? Was this discussed with them?

Reply:
No, this was just taken directly from the linked API docs as I did not see it documented anywhere that we had been granted additional permissions. Thanks for noting that, I will update the docstring here! That's good to know, as I was concerned we might be hitting the limit 😅